content/en/blog/2022/k8s-otel-expose.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Exposing an OpenTelemetry Collector for cross cluster communication
+linkTitle: Exposing an OpenTelemetry Collector
+date: 2022-08-25
+author: '[Benedikt Bongartz](https://github.com/frzifus)'
+spelling:
+  cSpell:ignore k8sattributes k8sattributesprocessor K8sattributes k8sprocessor
+  cSpell:ignore K8sprocessor KUBE
+---
+
+Exposing an [OpenTelemetry Collector](../../docs/collector/_index.md) currently requires a number of configuration steps. The goal of this blog post is to demonstrate "how to establish a secure communication" between two collectors in different kubernetes clusters.
+
```

Comment:
"how to establish a secure communication" could just be 'how to establish secure communication' and doesn't need the quotes